Coerce integers into Strings

cliveholloway opened the pull request:
Arrays of strings were not treating integers as strings in the JSON response. Appending an empty string coerces the value to a string.

Commit:
Coerce integers into Strings in lists

mohawk2 commented:
Actually on reflection could you add a test that fails without this? It'd be a pity to lose that behaviour by accident.

cliveholloway commented:
Will do - in next day or two.

Commit:
Add coerce of Integer ID into string
Add tests to confirm ID and String Types coerce to strings correctly when integers.

cliveholloway commented:
OK. tests added, and addition of coerce for ID type too.

mohawk2 commented:
Thank you!

mohawk2 commented:
Released as 0.53.
